Objective-C/CBLReplicator.mm

```diff
@@ -350,7 +364,7 @@ - (void) stop {
 - (void) _stop {
     [self cancelRetry];
     
-    if (_repl) {
+    if (_repl && _state != kCBLStateCreated) {
```

Comment:
what happens when calling stop on non starting _repl? is there anything we can get from _repl itself that it hasn't been started yet?

Reply:
it will crash, since the rawStatus.level after creating is connecting. It will fail somewhere [here](https://github.com/couchbaselabs/BLIP-Cpp/blob/8bbba50dcb3d023eb07df0f0649389e7c8c003a9/include/blip_cpp/LoopbackProvider.hh#L252)